### PR TITLE
Start Arabic Translation

### PR DESCRIPTION
**Pull request by 3laa-Khalil-Alarabi**

### CODE REVIEW

**Comment by bugron** [comment]
@3laa-Khalil-Alarabi thanks for the PR. Please, review our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), next time create a separate branch with an appropriate name (`fix/xxx` or `feature/xxx`). If this PR closes some issue, please, add to the description `Closes #XXXX` where `XXXX` is the issue number.
Also would you mind to continue translation of the rest of HTML5 and CSS challenges in this PR?

File: seed/challenges/html5-and-css.json [hunk 1]
```diff
@@ -31,6 +31,12 @@
       "descriptionCn": [],
       "nameFr": "",
       "descriptionFr": [],
+      "nameAr": "HTML قل مرحبا لعناصر",
+      "descriptionAr": [
+      "يمكنك تعديل <code>التعليمات البرمجية</code> في <code>محرر النصوص</code>،الذي قمنا بتضمينه داخل صفحة الويب هذه.",
+      "HTML ؟ هذا عنصر<code>&#60;h1&#62;Hello&#60;/h1&#62;</code>  هل ترى التعليمات البرمجية في محرر النصوص والتي تقول",
+      ],
+      "nameRu": "",
```
Comment (bugron) [inline seed/challenges/html5-and-css.json]: This is a duplicate of the line 40. Please, remove this line then do `git add .` and `git commit --amend` and then force push your branch to your fork: `git push -f`.

**Comment by bugron** [comment]
@3laa-Khalil-Alarabi when you finish your translation. please, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) all commits into one, thanks.

**Comment by bugron** [comment]
@3laa-Khalil-Alarabi what's your progress on this translation, are you still working on it?

**Comment by BerkeleyTrue** [comment]
This PR is now closed due to inactivity.

Please re-open when the PR is ready to be merged.

**Pull request closed without merging**